database/postgres/event.go

```diff
@@ -368,7 +368,7 @@ func (e *eventRepo) LoadEventsPaged(ctx context.Context, projectID string, filte
 		"query":           filter.Query,
 		"cursor":          filter.Pageable.Cursor(),
 		"idempotency_key": filter.IdempotencyKey,
-		"event_id":        filter.Query,
+		"event_id":        filter.EventID,
```

Comment:
@jirevwe, Can you take out this update pending Emmanuel's response so we can merge this pull request? I'm working on a larger filtering refactor at #1867